tests/unit/backend/corpora/api_server/mock_auth.py

```diff
@@ -144,6 +135,16 @@ def get_auth_token(app):
     return response.headers["Set-Cookie"]
 
 
+def make_token(token_claims: dict, token_duration: int = 0, additional_scope: list = None) -> dict:
+    addition_scope = additional_scope if additional_scope else []
```

Comment:
I'm not sure I like the rename to `addition_scope`. Confusing to me. Either reassign `additional_scope` or change to something visibly different.

Reply:
I think what you're saying is you'd prefer the name to be scope?

Reply:
actually, that's a typo.

Reply:
cool, you'll fix?